src/js/disability-benefits/526EZ/containers/ConfirmationPage.jsx

```diff
@@ -27,54 +27,79 @@ class ConfirmationPage extends React.Component {
   }
 
   render() {
-    const form = this.props.form;
-    const response = this.props.form.submission.response
-      ? this.props.form.submission.response.attributes
-      : {};
-    const name = form.data.relativeFullName;
-    const benefit = form.data.benefit;
+    const form = /* this.props.form */ initialData;
+    // const response = this.props.form.submission.response
+    // ? this.props.form.submission.response.attributes
+    // : {};
+    const name = form.veteranFullName;
+    const selectedDisabilities = form.disabilities; // Need to narrow this to selected disabilities
 
     return (
       <div>
-        <h3 className="confirmation-page-title">Claim received</h3>
-        <p>We usually process claims within <strong>30 days</strong>.</p>
-        <p>
-          We may contact you for more information or documents.<br/>
-          <i>Please print this page for your records.</i>
+        <h3 className="confirmation-page-title">Claim submitted</h3>
+        <p>We usually process claims within <strong>103 days</strong>.</p>
+        <p>We may contact you if we have questions or need more information. You can print this page for your records.</p>
+
+        <h4 className="confirmation-guidance-heading">
+          What happens after I apply?
+        </h4>
+        <p className="confirmation-guidance-message">
+          You don’t need to do anything unless we send you a letter asking for
+          more information. You can check the status of your claim online. The
+          time frame you see may vary based on how complex your claim is.
         </p>
+        <a href="/disability-benefits/track-claims">
+          Check the status of your disability claim
+        </a>
+        <br/>
+        <a href="/disability-benefits/after-you-apply/">
+          Learn more about what happens after you file a disability claim.
+        </a>
         <div className="inset">
-          <h4>Education Benefit Claim <span className="additional">(Form 21-526)</span></h4>
-          <span>for {name.first} {name.middle} {name.last} {name.suffix}</span>
+          <h4>
+            Disability Compensation Claim for Increase{' '}
```

Comment:
Oh curious. Is the `{' '}` to make sure there's a space? What if we keep the `<span>` on the same line?

Reply:
I'll fix that, good catch.